src/main/features/claim/routes/total.ts

```diff
@@ -9,13 +9,13 @@ import { DraftClaim } from 'drafts/models/draftClaim'
 
 /* tslint:disable:no-default-export */
 export default express.Router()
-  .get(Paths.totalPage.uri, (req: express.Request, res: express.Response, next: express.NextFunction) => {
+  .get(Paths.totalPage.uri, async (req: express.Request, res: express.Response, next: express.NextFunction) => {
```

Comment:
This looks cleaner to me, it had the same problems as above, mixing promises and async->await and extra unneeded API calls:
```
  .get(Paths.totalPage.uri,
    ErrorHandling.apply(async (req: express.Request, res: express.Response) => {
      const draft: DraftClaim = res.locals.claimDraft.document

      const interest: number = await interestAmount(draft)
      const totalAmount: number = draft.amount.totalAmount()

      const feeAmount: number = await FeesClient.calculateIssueFee(totalAmount + interest)
      res.render(Paths.totalPage.associatedView,
        {
          interestTotal: new TotalAmount(totalAmount, interest, feeAmount),
          interestClaimed: (draft.interest.type !== InterestType.NO_INTEREST)
        })
    }))
```

REQ